wallet/wallet.sol

```diff
@@ -337,7 +337,9 @@ contract Wallet is multisig, multiowned, daylimit {
     // and _data arguments). They still get the option of using them if they want, anyways.
     function execute(address _to, uint _value, bytes _data) external onlyowner returns (bytes32 _r) {
         // first, take the opportunity to check that we're under the daily limit.
-        if (underLimit(_value)) {
+        // we also must check that there is no data (this is not a contract invocation),
```

Comment:
For now I have changed this as @chriseth recommended without the length check. This should be safe and require more signers, which may be since the purpose of the wallet is mostly for multi-sig. 

We could also try a .send instead of a call, and if that fails then we could require the multi signer. I'm not sure how safe that is though.